### PR TITLE
Spelling, grammar, and punctuation corrections on README.md

### PR DESCRIPTION
**Pull request by cryptographicfool**
I noticed a few minor spelling, grammar and punctuation mistakes on the README.md file and decided to fix them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -824,7 +824,7 @@ Configure Monero with the -D SANITIZE=ON cmake flag, eg:
 cd build/debug && cmake -D SANITIZE=ON -D CMAKE_BUILD_TYPE=Debug ../..
 ```
 
-You can then run the monero tools normally. Performance will typically halve.
+You can then run the monero tools normally. The performance will typically halve.
```
Comment (moneromooo-monero) [inline README.md]: That one's incorrect.

File: README.md [hunk 2]
```diff
@@ -721,10 +721,10 @@ See [README.i18n.md](README.i18n.md).
 ## Using Tor
 
 > There is a new, still experimental, [integration with Tor](ANONYMITY_NETWORKS.md). The
-> feature allows connecting over IPv4 and Tor simulatenously - IPv4 is used for
+> feature allows connecting over IPv4 and Tor simultaneously - IPv4 is used for
 > relaying blocks and relaying transactions received by peers whereas Tor is
 > used solely for relaying transactions received over local RPC. This provides
-> privacy and better protection against surrounding node (sybil) attacks.
+> privacy and better protection against the surrounding node (sybil) attacks.
```
Comment (moneromooo-monero) [inline README.md]: That one's incorrect.

File: README.md [hunk 3]
```diff
@@ -421,8 +421,8 @@ application.
     cd monero
     ```
 
-* If you would like a specific [version/tag](https://github.com/monero-project/monero/tags), do a git checkout for that version. eg. 'v0.14.1.0'. If you don't care about the version and just want binaries from master, skip this step:
-	
+* If you would like a specific [version/tag](https://github.com/monero-project/monero/tags), do a git checkout for that version. eg. 'v0.14.1.0'. If you don't care about the version and just want binaries from the master, skip this step:
```
Comment (moneromooo-monero) [inline README.md]: That one's incorrect.

File: README.md (hunk 1 above)
Comment (cryptographicfool) [reply README.md]: Corrected!

File: README.md (hunk 2 above)
Comment (cryptographicfool) [reply README.md]: Corrected!

File: README.md (hunk 3 above)
Comment (cryptographicfool) [reply README.md]: Corrected!

**Comment by cryptographicfool** [comment]
Thank you for the review. I have made the corrections. 
Cheers!

**Comment by cryptographicfool** [comment]
Why do these build tests fail? I just updated README.md

**Comment by selsta** [comment]
@cryptographicfool They aren’t failing. monero-static-win32 isn’t setup correctly at the moment because we don’t do win32 builds.

**Comment by cryptographicfool** [comment]
> @cryptographicfool They aren’t failing. monero-static-win32 isn’t setup correctly at the moment because we don’t do win32 builds anymore.

@selsta  After going through the details and looking at other PRs, I realized the same. Thanks for clarifying. Cheers!

**Comment by dEBRUYNE-1** [comment]
@cryptographicfool - Could you squash the two commits into one?

**Comment by cryptographicfool** [comment]
> @cryptographicfool - Could you squash the two commits into one?


@dEBRUYNE-1  I have squashed the two commits, but as the previous commits are already in the pull request and have the changes have been verified by @moneromooo-monero ; I am having to use force push. I am not sure if that is the right thing to do, but I could be wrong.

If I should force push, I will do it right away. Otherwise, I am keeping these as it is for the moment. 

Thank you!

**Comment by selsta** [comment]
You can force-push.

**Comment by cryptographicfool** [comment]
> You can force-push.

Done. :)

**Comment by luigi1111** [comment]
@cryptographicfool looks like it'll need one more rebase.

**Comment by dEBRUYNE-1** [comment]
@cryptographicfool - ping.

**Comment by moneromooo-monero** [comment]
Rebased in https://github.com/monero-project/monero/pull/6788
This is thus now obsolete.

**Pull request closed without merging**